-----------------------------------------------------------------------------------------------------------------------------------
IDRIS WARNING: if you encounter performance issues regarding CPU multithreading, please consider loading one the following modules:
  tensorflow-gpu/py3/2.4.0-noMKL
  tensorflow-gpu/py3/2.5.0+nccl-2.8.3
  tensorflow-gpu version >= 2.6.0
-----------------------------------------------------------------------------------------------------------------------------------

Loading tensorflow-gpu/py3/2.4.1
  Loading requirement: gcc/8.5.0 cuda/10.2 nccl/2.6.4-1-cuda
    cudnn/8.0.4.30-cuda-10.2 openmpi/4.0.2-cuda
+ python -u byol_co_de.py
2024-10-17 15:49:40.999248: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.2
FUNCTIONS READY
Traceback (most recent call last):
  File "byol_co_de.py", line 79, in <module>
    data_gen = ByolGenerator(folder_path=folders, folder_extension=extensions, batch_size=256)
  File "/lustre/fswork/projects/rech/dnz/ull82ct/stage_astro/byol_generator.py", line 18, in __init__
    self.get_filename()
  File "/lustre/fswork/projects/rech/dnz/ull82ct/stage_astro/byol_generator.py", line 32, in get_filename
    for file_name in os.listdir(folder):
FileNotFoundError: [Errno 2] No such file or directory: '/lustre/fswork/projects/rech/kof/uve94ap/CUBES_HSC/PHOT/DEEP'
